apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page/entity-editor/history-section/get-history-events.ts

```diff
@@ -112,12 +111,12 @@ export const getHistoryEvents = (diffs: EntityDiff[], subgraph: Subgraph) => {
   for (const [index, [key, value]] of typedEntries(
     firstEntityEdition.properties,
   ).entries()) {
-    const propertyProvenance = firstEntityEdition.metadata.properties?.find(
-      /**
-       * @todo H-2775 – handle property objects and changes to array contents
-       */
-      (map) => map.path[0] === key,
-    );
+    /**
+     * @todo H-2775 – handle property objects and changes to array contents
+     */
+    const propertyProvenance = firstEntityEdition.propertyMetadata([
+      key,
+    ])?.provenance;
```

Comment:
This was probably not caused by this PR but the prior one, but just to flag I'm encountering an runtime error on the history:
![image](https://github.com/hashintel/hash/assets/42802102/2fb64ed2-98ac-4742-9626-db01522f3a44)

It seems that `getEntityRevision` is not actually returning instances of the `Entity` class, but the serialized version. When I log `firstEntityEdition` before the error is thrown I get this:
```JSON
{
    "firstEntityEdition": {
        "properties": {
           ...
        },
        "metadata": {
            ...
        }
    }
}
```

Reply:
Everything should now work again, thanks! 🙂